.release-please-manifest.json

```diff
@@ -1,6 +1,7 @@
 {
   "ansible": "1.0.0",
   "images/kairos-ubuntu": "0.1.3",
+  "images/vscode-ssh-server": "0.1.0",
```

Comment:
so this is the "current version" in this file, FWIW.  You probably want to omit this for now and set an `initial-version` in the package's config file.  You then need to remove that once it is released (and then I sort this file in that same commit).  It's slightly annoying, and we'd use an empty commit for this if this was not a monorepo.